pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -781,6 +781,14 @@ async def _sync_images(self):
         for record in records:
             if record['has_olm_bundle'] == '1' and record['status'] == '0' and record.get('nvrs', None):
                 operator_nvrs.append(record['nvrs'].split(',')[0])
+        for record in records:
+            # If failed build has dependents, remove it operator bundle from list because olm_bundle job won't succeed
+            if record['status'] != '0' and record.get('nvrs', None) and record.get('dependents', None):
+                distgit_nvrs = record['dependents'].split(', ')
```

Comment:
This var contains a list of distgit keys instead of nvrs. Better to use a different name.

Reply:
updated